### PR TITLE
[HWKMETRICS-748] remove alerts stuff

### PR DESCRIPTION
**Pull request by jsanda**

### CODE REVIEW

File: core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java [hunk 1]
```diff
@@ -103,7 +103,10 @@ public void initMethod() {
         for (TableMetadata tableMetadata : session.getCluster().getMetadata().getKeyspace(session.getLoggedKeyspace())
                 .getTables()) {
             if(tableMetadata.getName().startsWith(DataAccessImpl.TEMP_TABLE_NAME_PROTOTYPE)) {
-                session.execute(String.format("TRUNCATE %s", tableMetadata.getName()));
+                try {
+                    session.execute(String.format("TRUNCATE %s", tableMetadata.getName()));
+                } catch (Exception e) {
```
Comment (burmanm) [inline core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java]: Shouldn't be part of this PR?

File: core/metrics-core-service/src/test/java/org/hawkular/metrics/core/util/GCGraceSecondsManagerITest.java [hunk 2]
```diff
@@ -1,145 +0,0 @@
...
-                "'replication_factor': 2}");
-
-        manager.maybeUpdateGCGraceSeconds();
-
-        ResultSet resultSet = session.execute(
-                "select table_name, gc_grace_seconds from system_schema.tables where keyspace_name = '" + keyspace +
-                        "'");
-        resultSet.all().forEach(row -> {
-            if (row.getString(0).equals("data_compressed")) {
-                assertEquals(row.getInt(1), 0, "gc_grace_seconds for " + row.getString(0) + " is wrong");
```
Comment (burmanm) [inline core/metrics-core-service/src/test/java/org/hawkular/metrics/core/util/GCGraceSecondsManagerITest.java]: Why is this test removed?

File: core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java (hunk 1 above)
Comment (jsanda) [reply core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java]: What did I miss here?

File: core/metrics-core-service/src/test/java/org/hawkular/metrics/core/util/GCGraceSecondsManagerITest.java (hunk 2 above)
Comment (jsanda) [reply core/metrics-core-service/src/test/java/org/hawkular/metrics/core/util/GCGraceSecondsManagerITest.java]: It was already commented out. And since I knew it had been a source of problems and since I have been having lots of inconsistent test runs on travis, I decided to  altogether remove it. If you fixed it in your other PR, I am perfectly fine with bringing it back.

File: core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java (hunk 1 above)
Comment (burmanm) [reply core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java]: Added try {} catch parts.

Comment (jsanda) [reply core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java]: hmm... I have no idea why that change was made in this PR. I can revert the change.

**Comment by burmanm** [comment]
Last test failure was because master's fixes are not merged (for deleteMetric)
